retrofit/src/test/java/retrofit2/RetrofitTest.java

```diff
@@ -171,6 +171,37 @@
     assertSame(callFactory, two.callFactory());
   }
 
+  @Test public void builtInConvertersAbsentInCloneBuilder() {
+    Retrofit retrofit = new Retrofit.Builder()
+        .baseUrl(server.url("/"))
+        .build();
+
+    assertEquals(retrofit.converterFactories().size() - 1,
+        retrofit.newBuilder().converterFactories().size());
+  }
+
+  @Test public void builtInConvertersAddedOnBuild() {
```

Comment:
This feels like it’s testing an implementation detail?

Reply:
Basically anywhere you have `+ 1` tests something we don’t want to enforce. For example Moshi uses 5 built-in factories.

Reply:
This is probably implicitly tested wherever we test `Call<Void>`. I guess the right way to black box test that we don't regress is to install a custom converter for `Void` and validate that it's _not_ called.

Reply:
Agreed that this is testing an implementation detail. Removed this test in 4d4cee4c. 

Added Jake's suggested black box test in e5d30fc7.